model/task/task.go

```diff
@@ -91,16 +93,16 @@ type Task struct {
 	// Priority is a specifiable value that adds weight to the prioritization that task will be given in its
 	// corresponding distro task queue.
 	Priority int64 `bson:"priority" json:"priority"`
-	// PriorityRankValue is not persisted to the db, but stored in memory and passed to the task queue document.
-	// It is a mixture of Priority and the various task queue ranking factors that multiply on top of Priority.
-	PriorityRankValue int64  `bson:"priority_rank_value" json:"priority_rank_value"`
-	TaskGroup         string `bson:"task_group" json:"task_group"`
-	TaskGroupMaxHosts int    `bson:"task_group_max_hosts,omitempty" json:"task_group_max_hosts,omitempty"`
-	TaskGroupOrder    int    `bson:"task_group_order,omitempty" json:"task_group_order,omitempty"`
-	ResultsService    string `bson:"results_service,omitempty" json:"results_service,omitempty"`
-	HasCedarResults   bool   `bson:"has_cedar_results,omitempty" json:"has_cedar_results,omitempty"`
-	ResultsFailed     bool   `bson:"results_failed,omitempty" json:"results_failed,omitempty"`
-	MustHaveResults   bool   `bson:"must_have_results,omitempty" json:"must_have_results,omitempty"`
+	// SortingValueBreakdown is not persisted to the db, but stored in memory and passed to the task queue document.
+	// It contains information on what factors led to the overall queue ranking value for the task.
+	SortingValueBreakdown SortingValueBreakdown `bson:"sorting_value_breakdown" json:"sorting_value_breakdown"`
```

Comment:
could we actually just - the bson value to make it more clear that we won't be persisting?